chain-cli/src/keys.ts

```diff
@@ -30,11 +30,7 @@ export async function generateKeys(keysPath: string): Promise<void> {
   const devPrivateKey = secp.utils.bytesToHex(secp.utils.randomPrivateKey());
   const devPublicKey = secp.utils.bytesToHex(secp.getPublicKey(adminPrivateKey));
 
-  if (process.platform === "win32") {
-    execSync(`mkdir ${keysPath}`);
-  } else {
-    execSync(`mkdir -p ${keysPath}`);
-  }
+  execSync(`mkdir -p ${keysPath}`);
```

Comment:
I deliberately removed Windows support as it won't be neccessary after this one is merged: https://github.com/GalaChain/sdk/pull/67